src/main/java/org/broadinstitute/consent/http/enumeration/DatasetPropertyType.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.consent.http.enumeration;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+public enum DatasetPropertyType {
+    String,
+    Boolean,
+    Number,
+    Json,
+    Date;
```

Comment:
Let's make these all upper case with a lower cased string value for naming consistency. Having a lower cased type will also be closer to the actual jsonSchema values of type.

Reply:
only thing is that JDBI automatically uses Enum.name() instead of Enum.toString(). since name is final, it was not overridable, so I had to make a similar `getPropertyTypeAsString` function to get it to work. but i agree it is overall cleaner and closer to jsonSchema values!